packages/frontend/tests/integration/components/program-year/list-test.js

```diff
@@ -96,29 +96,29 @@ module('Integration | Component | program-year/list', function (hooks) {
     await component.newProgramYear.years.select(thisYear);
     await component.newProgramYear.done.click();
     const programYears = await this.owner.lookup('service:store').findAll('program-year');
-    const sortedProgramYears = sortBy(programYears.slice(), 'id');
+    const sortedProgramYears = sortBy(programYears, 'id');
```

Comment:
I'm a little bit worried about updating the tests and the application code in the same PR here. Are the changes linked? Or could the tests updates be in a second PR?

Reply:
Should be linked to the application code here: https://github.com/ilios/frontend/pull/8095/files#diff-12e203d3f8d49a7214eafaf3aa33d10d74239acd9d6ac6acc1bae17df2fc5256

Also, I changed tests alongside app code with my last de-slice()-ing, so... :D

Reply:
I accept this 🤠